src/platform/user/profile/vet360/components/AddressField/AddressField.jsx

```diff
@@ -63,6 +63,16 @@ export const convertNextValueToCleanData = value => {
   };
 };
 
+const validateZipCode = zipCode => {
```

Comment:
Validation got too tricky to do inline down on line 107 so I moved this to a new helper.